munit-scalafx/src/main/scala/munit/fx/ScalaFXSuite.scala

```diff
@@ -32,12 +33,14 @@ abstract class ScalaFXSuite extends FunSuite, ScalaFxAssertions:
      * @return
      *   The result of the test when a location can be pulled from given scope.
      */
-    def testFX[R <: Throwable, B](name: String)(body: Errors[R] ?=> A => B): Location ?=> Unit =
+    def testFX[R, F <: AssertionError: Typeable](name: String)(
```

Comment:
`TypeTest` (the thing aliased by this) is something I was unaware of. I like that it solves the generic pattern match issue without using `Manifest`, which is more complicated and has a wider interface. That's pretty cool.

Reply:
Thanks, Jack. Yeah, I didn't know about the existence of `TypeTest` and `Typeable` either. I ran into them while reading the Scala 3 documentation looking for alternatives to deal with generic types and the type erasure in the JVM 🙂